performance-tuning-practices.md

```diff
@@ -0,0 +1,439 @@
+---
+title: Performance Tuning Practices for OLTP Scenarios
+summary: This document describes how to analyze and tune performance for OLTP workloads.
+---
+
+# Performance Tuning Practices for OLTP Scenarios
+
+TiDB provides comprehensive performance diagnostics and analysis features, such as [Top SQL](/dashboard/top-sql.md) and [Continuous Profiling](/dashboard/continuous-profiling.md) features on the TiDB Dashboard, and TiDB [Performance Overview Dashboard](/grafana-performance-overview-dashboard.md).
+
+This document describes how to use these features together to analyze and compare the performance of the same OLTP workload in seven different runtime scenarios, which demonstrates a performance tuning process to help you analyze and tune TiDB performance quickly.
+
+> **Note:**
+>
+> [Top SQL](/dashboard/top-sql.md) and [Continuous Profiling](/dashboard/continuous-profiling.md) are not enabled by default so you need to enable them in advance.
+
+By running the same application with different JDBC configurations in these scenarios, this document shows you how the overall system performance is affected by different interactions between applications and databases so that you can apply [Best Practices for Developing Java Applications with TiDB](/best-practices/java-app-best-practices.md) for better performance.
+
+## Environment description
+
+This document takes an OLTP simulation workload of a bank transaction system for demonstration. The configurations of the simulation environment are as follows:
+
+- Application development language for the workload: JAVA
+- SQL statements used in business: 200 statements in total, 90% of which are SELECT statements. It is a typical read-intensive OLTP scenario.
```

Comment:
It is a typical read-intensive OLTP scenario----->It is a typical read-heavy OLTP workload ?